nms/app/views/subscriber/SubscriberUtils.ts

```diff
@@ -120,49 +114,58 @@ export function validateSubscribers(
   subscribers: Array<SubscriberInfo>,
   action: SubscriberActionType,
 ) {
-  const errors: {
-    [error: SubscriberError]: Array<number>,
-  } = {};
-  const imsiList = [];
+  const errors: Record<SubscriberError, Array<number>> = {};
+  const imsiList: Array<string> = [];
 
   Object.keys(SUBSCRIBER_ADD_ERRORS).map(error => {
-    const subscriberError = SUBSCRIBER_ADD_ERRORS[error];
+    const subscriberError =
+      SUBSCRIBER_ADD_ERRORS[error as keyof typeof SUBSCRIBER_ADD_ERRORS];
```

Comment:
We should use the extracted type `SubscriberError` instead of repeating this

Reply:
There is the actual value used here, not only the type.